fda/basis.py

```diff
@@ -738,6 +738,8 @@ def __init__(self, domain_range=None, nbasis=None, order=4, knots=None):
         else:
             knots = list(knots)
             knots.sort()
+            if domain_range is None:
+                domain_range = (0, 1)
```

Comment:
It  should be:
```suggestion
                domain_range = (knots[0], knots[-1])
```
or I am missing something?